Views/IFretBoard.cs

```diff
@@ -0,0 +1,4 @@
+namespace Fretty.Views;
+
+public interface IFretBoard
```

Comment:
Can we leave the

`void DrawChord(IEnumerable<int[]> coordinates);`

and maybe just `throw new NotImplemented()`?

Just because that is the endpoint that other parts of the code are going to rely on being able to use.
We are definitely going to want an implementation for this soon.

Reply:
I gotchu